app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,32 @@
+class FoodsController < ApplicationController
+  def index
+    @foodlist = current_user.foods
+  end
+
+  def new
+    @food = Food.new
+  end
+
+  def create
+    @food = Food.new(food_params)
+    @food.user_id = current_user.id
+
+    if @food.save
+      redirect_to foods_path, notice: 'Food was successfully created.'
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    @food = Food.find(params[:id])
+    @food.destroy
+    redirect_to foods_path, notice: 'Successfully destroyed foodlist.'
+  end
+
```

Comment:
- [ ] @Team it is good practice to create a  success and fail paths when you create methods such as destroy, this enables easier troubleshooting in case of an error.